ssh/session.go

```diff
@@ -201,17 +201,22 @@ func (s *Session) connect(passwd string, key *rsa.PrivateKey, session sshserver.
 		go func() {
 			buf := make([]byte, 1024)
 			n, err := stdout.Read(buf)
-			waitingString := ""
 			if err == nil {
-				waitingString = string(buf[:n])
 				c.RecordSession(&models.SessionRecorded{
 					UID:     s.UID,
-					Message: waitingString,
+					Message: string(buf[:n]),
```

Comment:
Please, rework the commit log so it explains what we are fixing here and the motivation for the change.

It must follow the pattern:

```
scope: short description

long description
```

For the scope, it seems `ssh` is a good fit ;-)

Reply:
Thanks, I changed it.